ui/react-ui/src/components/routes/create/Create.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useRef } from 'react';
+import Form from 'react-bootstrap/Form'
+import Button from 'react-bootstrap/Button';
+import Alert from 'react-bootstrap/Alert';
+import './Create.css';
+
+const Create = () => {
+
+    const [firstName, setFirstName] = useState();
+    const [lastName, setLastName] = useState();
+    const [email, setEmail] = useState();
+    const [validated, setValidated] = useState();
+    const [userCreateSuccess, setUserCreateSuccess] = useState(false);
+    const [isVisible, setIsVisible] = useState(false);
+
+    const formRef = useRef(null);
+
+    const handleChange = (e) => {
+        const fieldName = e.target.name;
+        const fieldValue = e.target.value;
+
+        // Save fields to state
+        if (fieldName === 'firstName') {
+            setFirstName(fieldValue);
+        } else if (fieldName === 'lastName') {
+            setLastName(fieldValue);
+        } else if (fieldName === 'email') {
+            setEmail(fieldValue);
+        }
+    }
+
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        const form = formRef.current;
+
+        // Form is invalid
+        if (form.checkValidity() === false) {
+            setValidated(true);
+        } 
+        // Form is good to go
+        else {
+            await createUser(); // Call function to create user
+            form.reset();
+        }
+    }
+
+    const createUser = async () => {
+        const body = {firstName: firstName, lastName: lastName, email: email};
```

Comment:
You can use shorthand here for this object, if the key name and the variable name are the same you can just put:
{ firstName, lastName, email }

Reply:
Thanks for the tip Kevin, I'll be sure to test it out and incorporate the new syntax!